### ISSUE TITLE
PI: Visual Feedback shows identical values for Best and Current

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#436]
In the Viesual Feedback of a PI, the values of parameters shown in the columns "Best" and "Current" are always equal, though the error might be different.

![image](https://user-images.githubusercontent.com/25903040/72983096-d98bae00-3de0-11ea-9e04-a3049e404381.png)

**Comment by msevestre** [comment]
@PavelBal Always equal? Like all the time in any project? Or just on this specific case?

**Comment by PavelBal** [comment]
At least 100% of the time when I look at the visual output :) Any project, any scenario.

Also probably connected to this one (and made me swear and cry a lot) - when aborting the PI and transferring the results, the values of the last iteration and not the best values are transferred.

**Comment by msevestre** [comment]
What??? 

That's only when you abort a PI? How long have you been sitting on this one?

**Comment by PavelBal** [comment]
Yes, only when aborting (not sure if it is always or only in some cases). I made a habbit not to abort the PI, but yes, it is a very painful "feature".

**Comment by msevestre** [comment]
Well if we don't know about the bug... we won't correct it. Thanks for reporting it :)

**Comment by msevestre** [comment]
I can confirm problem #1 in PKSim
![image](https://user-images.githubusercontent.com/1041237/72990415-1a260000-3dbe-11ea-8523-562fe107d75f.png)
